src/Compilers/Core/Portable/Emit/EditAndContinue/DeletedMethodBody.cs

```diff
@@ -68,15 +68,34 @@ public DeletedMethodBody(DeletedMethodDefinition methodDef, EmitContext context)
 
         public StateMachineStatesDebugInfo StateMachineStatesDebugInfo => default;
 
-        private static ImmutableArray<byte> GetIL(EmitContext context)
+        private static ImmutableArray<byte> GetIL(IMethodDefinition methodDef, IMethodReference overriddenMethod, uint overriddenMethodToken, EmitContext context)
         {
             var missingMethodExceptionStringStringConstructor = context.Module.CommonCompilation.CommonGetWellKnownTypeMember(WellKnownMember.System_MissingMethodException__ctor);
             Debug.Assert(missingMethodExceptionStringStringConstructor is not null);
 
             var builder = new ILBuilder((ITokenDeferral)context.Module, null, OptimizationLevel.Debug, false);
-            builder.EmitOpCode(System.Reflection.Metadata.ILOpCode.Newobj, 4);
-            builder.EmitToken(missingMethodExceptionStringStringConstructor.GetCciAdapter(), context.SyntaxNode!, context.Diagnostics);
-            builder.EmitThrow(isRethrow: false);
+
+            // If we're deleting a method that overrides a base method then we trampoline to it,
```

Comment:
>If we're deleting a method that overrides a base method then we trampoline to it, [](http://example.com/codeflow?start=15&length=81)

How important is this scenario? In short, why not just throw a MissingMethodException when deleting an override?

Reply:
It just makes things a lot nicer for users. eg, if I had 3 virtual methods in a chain, each one calling `base`, and I remove the middle layer, why should a call to the highest layer throw? Or code that operates on a base class throw if I change something in a child class?

Reply:
Another example of where trampolining would be helpful, though not directly solved by this PR, would be https://github.com/dotnet/roslyn/issues/65442, where its not obvious to the user than an edit they made to a method signature, causes a new method to be emitted